### ISSUE TITLE
[Feature Request] Siri support

### ISSUE THREAD

**Issue by numselli** [LorenzoPane/EvilScheme#16]
**Describe the bug**
EvilScheme does not work when using siri 

**To Reproduce**
1) activate siri
2) say "get directions to the closest movie theatre"
3) opens directions in apple maps not google maps

1) activate siri
2) say "show me photos of dogs"
3) tap result 
4) it opens in safari not brave

**Expected behavior**
1) opens directions in google maps 
2) opens link from siri in brave

**Extra Info**
1. Device model:  iPhone 6s
2. iOS version:  13.3.1
3. Evil Scheme version:  0.1.1

**Comment by LorenzoPane** [comment]
Yup. Siri support is on my todo list. I had it working pretty unreliably before release, so I decided to disable it for now.

**Comment by squintgit** [comment]
> Yup. Siri support is on my todo list. I had it working pretty unreliably before release, so I decided to disable it for now.

When you add it I would love to be able to set my default music app so I could say “Play such and such” and it does so in your default music app which isn’t the stock Music app, so we don’t have to say “Play such and such in Spotify” or “Play such and such in YouTube Music” etc

**Comment by Void48** [comment]
Is there any update on this?
